src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const RentalCost = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return RentalCost - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
We don't need `else if` here. If first conditional on line 14 is true, this code would never be executed because of return. Refer to item 5 of task checklist.